docs/source/guide/ddd-4-low-level.myst

```diff
@@ -12,3 +12,163 @@ kernelspec:
 ---
 
 # What happens when I use DDD?
+
+The workflow of Digital Dynamical Decoupling (DDD) in Mitiq is represented in the figure below.
+
+```{figure} ../img/ddd_workflow.png
+---
+width: 700px
+name: ddd-workflow
+---
+Workflow of the DDD technique in Mitiq.
+```
+
+- The user provides a `QPROGRAM`, (i.e. a quantum circuit defined via any of the supported frontends).
```

Comment:
Might be good to add a link to `/frontends-backends.myst`.

Reply:
Good idea! Done: https://github.com/unitaryfund/mitiq/pull/1276/commits/d588fbcc416ef358520e9fe47ea5e6e083225042